### PR TITLE
style: fix large Input size

### LINKED ISSUE #38659: ant design5.0 组合输入框的按钮和输入框边框不对齐
### Reproduction link

[https://next.ant.design/components/input-cn](https://next.ant.design/components/input-cn)

### Steps to reproduce

1. 进入 https://next.ant.design/components/input-cn
2. 在搜索框介绍里面，大型的组合输入框的按钮与输入框边框没有对齐
image

### What is expected?

应该按钮与输入框应该边框对齐

Ant Design4.0 是对齐的
image

Ant Design5.0 没对齐
image

### What is actually happening?

Ant Design5.0 组合输入框，按钮与输入框边框没有对齐

| Environment | Info |
| --- | --- |
| antd | 5.0 |
| React | xxx |
| System | xxx |
| Browser | edge |

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #38659

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  -         |
| 🇨🇳 Chinese | -          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/style/index.tsx [hunk 1]
```diff
@@ -854,7 +854,7 @@ export function initInputToken<T extends GlobalToken = GlobalToken>(token: T): I
       3,
     ),
     inputPaddingVerticalLG:
-      Math.ceil(((token.controlHeightLG - token.fontSizeLG * token.lineHeightLG) / 2) * 10) / 10 -
+      Math.ceil(((token.controlHeightLG - token.fontSizeLG * token.lineHeight) / 2) * 10) / 10 -
```
[the remark this answers is not in the document]
Comment (afc163) [reply components/input/style/index.tsx:857]: 原因是：https://github.com/ant-design/ant-design/issues/38659#issuecomment-1319471556

[the remark this answers is not in the document]
Comment (yoyo837) [reply components/input/style/index.tsx:857]: #38435

**Review by li-jia-nan — APPROVED** [review]

File: components/input/style/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MadCcc) [reply components/input/style/index.tsx:857]: 改成 round 或者 floor 试试？

File: components/input/style/index.tsx [hunk 2]
```diff
@@ -752,7 +752,7 @@ const genSearchInputStyle: GenerateStyle<InputToken> = (token: InputToken) => {
       // fix slight height diff in Firefox:
       // https://ant.design/components/auto-complete-cn/#components-auto-complete-demo-certain-category
       [`${componentCls}-lg`]: {
-        lineHeight: token.lineHeight - 0.0002,
+        lineHeight: token.lineHeightBG - 0.0002,
```
Comment (MadCcc) [inline components/input/style/index.tsx:755]:
```suggestion
        lineHeight: token.lineHeightLG - 0.0002,
```

Comment (MadCcc) [reply components/input/style/index.tsx:755]: 居然有魔法

**Pull request merged**